app/models/concerns/database_indexable.rb

```diff
@@ -42,4 +42,12 @@ def generate_searchable_content # rubocop:disable Metrics/AbcSize,Metrics/Method
       ],
     ).tsvector
   end
+
+  private
+
+  def unique_words(text)
+    return if text.nil?
+
+    text.downcase.scan(/\w+/).uniq
```

Comment:
Niche thing: I think `\w` doesn't include dashes - not sure whether or not this is a big deal. Perhaps `[\w-]+`?

Reply:
@csutter @elliotcm looking at what people search for they don't search for things using dashes. But we do have the occasional use of a dash e.g. in a job title like "Part-time class teacher" etc. So it might matter for that?

Reply:
@csutter Updated.